configuration/group_vars/all.yml

```diff
@@ -11,6 +11,8 @@ consul_server_config_dir: "{{consul_config_dir}}/server"
 consul_client_config_dir: "{{consul_config_dir}}/client"
 consul_data_dir: "/opt/consul"
 vault_config_dir: "/etc/vault.d"
+consul_tls_dir: "{{consul_config_dir}}/tls"
+vault_tls_dir: "{{vault_config_dir}}/tls"
 
-local_binaries: true
+local_binaries: false
 local_enterprise_binary_dir: "/usr/local/hashi-enterprise"
```

Comment:
Do we want to make this relative to local path?  One needs sudo privs just to put the binaries there, thus requiring the implementer to be an admin to stand a cluster up in the default configuration.

Reply:
I did not want to in the repo path to avoid any chance of it getting checked in.
We can move it to a non-protected folder, I did that to avoid me deleting them easily on my machine.